packages/@datapull/http-api-origin/index.js

```diff
@@ -26,10 +26,85 @@ class HttpApiOrigin {
     };
   }
 
+  async customAuth(config) {
+    return new Promise((resolve, reject) => {
+      if (config.auth !== 'custom') {
+        return resolve();
```

Comment:
Did you mean `reject()` here?

Reply:
`resolve()` is correct here - it means no "custom" auth is needed, we can proceed to an actual request